packages/fx-core/src/common/azure-hosting/interface.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export enum HostType {
+  AppService = "webapp",
```

Comment:
Maybe we could check it later if we use webapp or appservice as value of hostType to keep concept consistent